### PR TITLE
Add Awesome Prometheus collection of related resources

### PR DESCRIPTION
**Pull request by avivace**
Signed-off-by: Antonio Vivace <avivace4@gmail.com>

### CODE REVIEW

File: content/docs/introduction/media.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ sort_rank: 7
 There is a [subreddit](https://www.reddit.com/r/prometheusmonitoring)
 collecting all Prometheus-related resources on the internet.
 
+[Awesome Prometheus](https://github.com/roaldnefs/awesome-prometheus) is a  curated list of related blog posts, projects and tools.
```
Comment (beorn7) [inline content/docs/introduction/media.md]: Nit: We usually use the Oxford comma (put a comma before "and").

**Comment by avivace** [comment]
Yes, highlighting the community aspect of it makes more sense

**Comment by beorn7** [comment]
@avivace So would you like to incorporate my suggestion?

**Comment by avivace** [comment]
@beorn7 Done.

**Comment by brian-brazil** [comment]
I'm not sure why this is in brackets. I'd go for something like "Awesome Prometheus contains a more comprehensive community-maintained list of resources."

We may want to look at cleaning down this page too.

**Comment by beorn7** [comment]
Sure, fine with me as well.

**Comment by beorn7** [comment]
@avivace could you modify according to @brian-brazil 's suggestion? (Not the "cleaning down" part, I understand that as a general note, not relevant to this PR in particular.)

Alternatively, if @avivace is not responsive, I'd suggest to merge this and change the detailed wording right after it. This is a helpful and simple addition, and it is sad that it takes almost three months to get it live.

**Comment by beorn7** [comment]
I have created #1590 to incorporate @brian-brazil 's last comment and wrap this up.

**Pull request closed without merging**